Lib/test/test_subprocess.py

```diff
@@ -1360,6 +1360,27 @@ def test_communicate_epipe(self):
         self.addCleanup(p.stdin.close)
         p.communicate(b"x" * 2**20)
 
+    def test_string_representation(self):
+        """Test stirng representation Popen object without returncode."""
+        code = 'import sys; sys.exit(1)'
+        cmd = [sys.executable, '-c', code]
+        with subprocess.Popen(cmd) as proc:
```

Comment:
Would be cool to check the same proc after it was closed and see the `repr` change.